examples/tidal_barrage/modules/input_barrages.py

```diff
@@ -0,0 +1,55 @@
+from modules.parameterisations import *
+
+
+def initialise_barrage(Barrage_Number=1):
```

Comment:
(Dummy) arguments should in principle be lower case. Exceptions are for instance mathematical (like Q_s) which are absolutely fine.